cloudigrade/account/views.py

```diff
@@ -7,6 +7,7 @@
 
 
 class AccountViewSet(mixins.CreateModelMixin, viewsets.ReadOnlyModelViewSet):
+    # pylint: disable=too-many-ancestors
```

Comment:
Given that using multiple mixins is kinda _by design_ with DRF, I'm inclined to throw this to the rc file instead of here, but I guess this is OK for now if it's the only instance.

Reply:
So far this was the only occurrence, I think if it shows up more times and keeps being a false positive then yea, it belongs permanently disabled.